EdvClient.js

```diff
@@ -13,35 +13,38 @@ const DEFAULT_CHUNK_SIZE = 1048576;
 
 export class EdvClient {
   /**
+   * @typedef https
+   *
    * Creates a new EdvClient instance. An EDV is an Encrypted Data Vault.
    *
    * In order to support portability (e.g., the use of DID URLs to reference
-   * documents), Encrypted Data Vault storage MUST expose an HTTPS API with a
-   * URL structure that is partitioned like so:
+   * documents), Encrypted Data Vault storage MUST expose an HTTPS API with
+   * a URL structure that is partitioned like so:
    *
    * <edvID>/documents/<documentID>
    *
    * The <edvID> must take the form:
    *
    * <authority>/edvs/<random multibase base58 encoded ID>
    *
-   * @param {Object} options - The options to use.
-   * @param {string} [id=undefined] - The ID of the EDV that must be a URL
-   *   that refers to the EDV's root storage location; if not given, then
+   * @param {object} options - The options to use.
+   * @param {string} [options.id=undefined] - The ID of the EDV that must be a
+   *   URL that refers to the EDV's root storage location; if not given, then
    *   a separate capability must be given to each method called on the client
    *   instance.
-   * @param {function} [keyResolver=this.keyResolver] - A default function that
-   *   returns a Promise that resolves a key ID to a DH public key.
-   * @param {Object} [keyAgreementKey=null] - A default KeyAgreementKey API for
-   *   deriving shared KEKs for wrapping content encryption keys.
-   * @param {Object} [hmac=null] - A default HMAC API for blinding indexable
-   *   attributes.
-   * @param {https.Agent} [httpsAgent=undefined] - An optional HttpsAgent to
-   *   use to handle HTTPS requests.
-   * @param {Object} [defaultHeaders=undefined] - An optional defaultHeaders
-   *   object to use with HTTP requests.
+   * @param {Function} [options.keyResolver=this.keyResolver] - A default
+   *   function that returns a Promise that resolves a key ID to a DH public
+   *   key.
+   * @param {object} [options.keyAgreementKey=null] - A default KeyAgreementKey
+   *   API for deriving shared KEKs for wrapping content encryption keys.
+   * @param {object} [options.hmac=null] - A default HMAC API for blinding
+   *   indexable attributes.
+   * @param {https.Agent} [options.httpsAgent=undefined] - An optional
```

Comment:
We are not importing node's `https` library and for good reason (it's backend only). So if you are ok with the change in your typedef above then replace `@param {https.Agent}` with `@param {httpsAgent}`